src/realm/sync/network/default_socket.cpp

```diff
@@ -3,18 +3,21 @@
 #include <realm/sync/network/network.hpp>
 #include <realm/sync/network/network_ssl.hpp>
 #include <realm/sync/network/websocket.hpp>
+#include <realm/util/random.hpp>
 
 namespace realm::sync::websocket {
 
 namespace {
-class EZSocketImpl final : public EZSocket, public Config {
+class DefaultWebSocketImpl final : public DefaultWebSocket, public Config {
```

Comment:
can't we use WebSocketInterface directly already? They are almost identical and we have to make the change at some point anyway.

Reply:
I hadn't updated the use of WebSocket in `ClientImpl::Connection`, since I was waiting to do this in #6094, but I went ahead and updated these.